doc/train/finetuning.md

```diff
@@ -45,3 +47,121 @@ To obtain a more simplified script, for example, you can change the {ref}`model
         "fitting_net" : {}
     }
 ```
+
+## PyTorch Implementation {{ pytorch_icon }}
+
+In PyTorch version, leveraging the flexibility offered by the framework and the multi-task training capabilities provided by DPA2 [paper](https://arxiv.org/abs/2312.15492),
+we have introduced an updated, more adaptable approach to fine-tuning. This methodology encompasses two primary variations:
+
+### 1. Single-task fine-tuning
+
+This fine-tuning method is similar to the fine-tune approach supported by TensorFlow.
+It utilizes a standard pre-trained model (`pretrained.pt`, single-task pre-trained) and modifies the energy bias within its fitting net before continuing with training.
+The command for this operation is:
+
+```bash
+$ dp --pt train input.json --finetune pretrained.pt
```

Comment:
how to finetune from a randomly initialized fitting net?

Reply:
We do not yet support fine-tuning from a randomly initialized fitting net in single-task fine-tuning from single-task pretrained model, which is the same as implementations in TensorFlow. I've opened an [issue](https://github.com/deepmodeling/deepmd-kit/issues/3741) for this.